Assets/HealthTracker.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class HealthTracker : MonoBehaviour {
+
+    GameObject healthBar;
+    GameObject staminaBar;
```

Comment:
Separate Health and Stamina. Some enemies that have health won't have stamina. SInce health and stamina have so much in common, consider making a base class that handles the stat bar management.